test/acceptance/PushPullDBInsertedNotificationsSpec.scala

```diff
@@ -127,21 +125,16 @@ class PushPullDBInsertedNotificationsSpec extends AcceptanceTestSpec
   }
 
   private def callAPIAndMakeSureItReturns202(requestToAPI: FakeRequest[AnyContentAsXml]) = {
-    val result: Option[Future[Result]] = route(app, requestToAPI)
-
-    status(result.value) shouldBe ACCEPTED
-
+    status(route(app, requestToAPI).value) shouldBe ACCEPTED
   }
 
   private def callRemoteAPIAndMakeSureItReturns202(requestToAPI: FakeRequest[AnyContentAsXml]) = {
-
     implicit val hc = HeaderCarrier()
     val body = requestToAPI.body.asXml.get.toString()
     val headers = requestToAPI.headers.headers
     val result: HttpResponse = await(httpClient.POSTString("http://192.168.160.60:9821/customs-notification/notify", body, headers))
```

Comment:
should replace hard-wired ip address with some text like `set-remote-ip-address`